@hubblecommerce/hubble/core/components/swComponents/slots/image-gallery-slot.vue

```diff
@@ -118,14 +124,15 @@ export default {
             const i = slider.currentSlide;
 
             this.activeImageIndex = i;
+            this.hooperData = slider;
 
             if (this.$refs.preview) this.$refs.preview.slideTo(this.activeImageIndex);
         },
     },
 };
 </script>
 
-<style lang="scss" scoped>
+<style lang="scss">
```

Comment:
why this has to be no longer scoped?

Reply:
If its scoped some styles are not applied, e.g. `.image-gallery img`
That is transpiled to `.image-gallery img[data-v-7cb6626c]`